op-challenger/scripts/alphabet/mallory.sh

```diff
@@ -1,22 +1,28 @@
 #!/bin/bash
 set -euo pipefail
 
-DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
-cd "$DIR"
+SOURCE_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+CHALLENGER_DIR=$(echo ${SOURCE_DIR%/*/*})
 
-DISPUTE_GAME_PROXY="0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e"
+# Check that the fault game address file exists
+FAULT_GAME_ADDR_FILE="$CHALLENGER_DIR/.fault-game-address"
+if [[ ! -f "$FAULT_GAME_ADDR_FILE"]]; then
+    echo "Game not initialized, exiting..."
+    exit 1
+fi
 
+DISPUTE_GAME_PROXY="0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e"
```

Comment:
This depends on the order of the deploy script sending transactions? Could you get this from `.devnet/addresses.json`? Perhaps could check that `cast code` isn't `0x` to catch errors faster?

Reply:
Actually not used, removed. Will follow up in another pr to add the preimageoracle address in there.